patches/pkg-entry-points+1.0.2.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/node_modules/pkg-entry-points/.DS_Store b/node_modules/pkg-entry-points/.DS_Store
```

Comment:
This package provides some useful features for extracting the package.json exports, especially handling globstar patterns, but it only provides an async API, which we cannot use in the sync context where we call this. This patch is adding a sync variant, which we should get rid off again before merging, once get to land this upstream: https://github.com/privatenumber/pkg-entry-points/issues/4